generators/skill/templates/function/python2/src/__main__.py

```diff
@@ -6,11 +6,15 @@ def main(req):
     try:
         result = main_impl(req['payload'])
     except KeyError as e:
-        result = { 'error': 'Missing required key {0}'.format(e.args[0]) }
-    return { 'payload': result }
+        result = {'error': 'Missing required key {0}'.format(e.args[0])}
+    return {'payload': result}
+
 
 def main_impl(payload):
     # unpack all inputs first
-    text = payload['text']
```

Comment:
You can also replace the if/else check (lines 15-18) with the one-liner: `text = payload.get('text', "(silence)")`